modules/transactions.js

```diff
@@ -644,39 +646,51 @@ Transactions.prototype.onBind = function (scope) {
  * @see {@link http://apidocjs.com/}
  */
 Transactions.prototype.internal = {
-	getTransactionHistory: function(req, cb) {
-
-		//let trsDate=[],trsCount=[];
-
-		let  fortnightBack = new Date(+new Date - 12096e5);
-
-		fortnightBack.setHours(0,0,0,0);	
-
-		let startTimestamp = slots.getTime(fortnightBack);
-		
-		let endDate = new Date(+new Date - (60 * 60 * 24 * 1000));
-		
-		endDate.setHours(0,0,0,0);
-		
-		let endTimestamp = slots.getTime(endDate);
-
-		library.db.query(sql.getTransactionHistory, {
-			startTimestamp: startTimestamp + epochTime,
-			endTimestamp: endTimestamp + epochTime,
-			epochTime: epochTime
-		})
-		.then(function(trsHistory) {
-
-			//for(let i=0;i<trsHistory.length;i++) {
-			//	trsDate[i] = new Date(trsHistory[i].time).toDateString();
-			//	trsCount[i] = trsHistory[i].created;
-			//}
-
-			return setImmediate(cb, null, {success: true, trsData: trsHistory });
-		})
-		.catch(function(err) {
-			return setImmediate(cb, {success: false, err: err});
-		});
+	getTransactionHistory: function (req, cb) {
```

Comment:
Is it API method using only for explorer?

Reply:
Yes, It is used in Explorer only.